bin/node/runtime/src/lib.rs

```diff
@@ -140,6 +140,7 @@ impl frame_system::Trait for Runtime {
 	type Version = Version;
 	type ModuleToIndex = ModuleToIndex;
 	type AccountData = pallet_balances::AccountData<Balance>;
+	type MigrateAccount = (Balances, Identity, Democracy, Elections, ImOnline, Recovery, Session, Society, Staking, Vesting);
```

Comment:
Note for people using this as inspiration for migration:
This order is incorrect because the account migration for `Session` depends on the migration for `Staking`. It thus needs to be e.g. `... Recovery, Staking, Session, Society, ...`